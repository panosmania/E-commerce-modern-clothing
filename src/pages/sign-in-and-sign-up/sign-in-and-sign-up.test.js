import { shallow } from "enzyme";
import React from "react";
import toJson from "enzyme-to-json";

import { SignInAndSignUpPage } from "./sign-in-and-sign-up.component";

describe("SignInAndSignUpPage component", () => {
  it("should render MenuItem component", () => {
    const wrapper = shallow(<SignInAndSignUpPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
