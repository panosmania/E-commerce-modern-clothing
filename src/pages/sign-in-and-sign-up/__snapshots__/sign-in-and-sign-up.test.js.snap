// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInAndSignUpPage component should render MenuItem component 1`] = `
<div
  className="sign-in-and-sign-up"
>
  <Connect(SignIn) />
  <Connect(SignUp) />
</div>
`;
